dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/DateUtils.java

```diff
@@ -353,6 +353,9 @@ public static String format2Duration(Date start, Date end) {
         if (end == null) {
             end = new Date();
         }
+        if (start.after(end)) {
```

Comment:
This is an unexpected case, we need to log a warning here.

Reply:
done